etc/migration/9.2-to-10.0/migration.cql

```diff
@@ -0,0 +1 @@
+ALTER "Principals" ADD "guest" boolean;
```

Comment:
Yes, that makes sense. Would it be helpful for the code to distinguish between guest users (ie. users on the guest tenant) and invited users (ie. users invited by email to a normal tenant) ?